frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts

```diff
@@ -1,36 +1,33 @@
 import * as _ from 'lodash';
+import i18n from '@console/internal/i18n';
 import { K8sResourceKind } from '@console/internal/module/k8s';
 import { ClusterServiceVersionKind } from './types';
 import { referenceForProvidedAPI, providedAPIsFor } from './components';
 import * as operatorLogo from './operator.svg';
 
-export const isInternal = (crd: { name: string }): boolean => {
-  const internalOpListString = _.get(
-    crd,
-    ['csv', 'metadata', 'annotations', 'operators.operatorframework.io/internal-objects'],
-    '[]',
-  );
+// FIXME This is a duplicate helper function. See utils.ts
+export const isInternal = (crd): boolean => {
+  const internalOpListString =
+    crd?.csv?.metadata?.annotations?.['operators.operatorframework.io/internal-objects'] ?? '[]';
   try {
     const internalOpList = JSON.parse(internalOpListString); // JSON.parse fails if incorrect annotation structure
     return internalOpList.some((op) => op === crd.name);
   } catch {
     /* eslint-disable-next-line no-console */
-    console.error('Failed to parse CSV annotation: Invalid JSON structure');
+    console.error(i18n.t('olm~Failed to parse CSV annotation: Invalid JSON structure'));
```

Comment:
We don't need to localize strings that are only printed to the JS console. (It might be better not to in fact since these are meant for developers to debug problems.)